presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -120,6 +120,13 @@ Map Functions
 
         SELECT map_top_n_keys(map(ARRAY['a', 'b', 'c'], ARRAY[1, 2, 3]), 2, (x, y) -> IF(x < y, -1, IF(x = y, 0, 1))) --- ['c', 'b']
 
+.. function:: map_top_n_keys_by_value(x(K,V), n) -> array(K)
+
+    Returns top ``n`` keys in the map ``x`` by sorting its values in descending order. If two or more keys have equal values, the higher key takes precedence.
+    ``n`` must be a non-negative integer.
+
+        SELECT map_top_n_keys_by_value(map(ARRAY['a', 'b', 'c'], ARRAY[2, 1, 3]), 2) --- ['c', 'a']
```

Comment:
Very strange - I thought I had a comment here asking to name it better - MAP_KEYS_BY_TOP_N_VALUES

Reply:
Hmm...probably didn't got saved earlier. `MAP_KEYS_BY_TOP_N_VALUES ` doesn't feel like its gonna return N keys.